agences-regionales-sante/martinique/2020-03-23.yaml

```
@@ -0,0 +1,11 @@
+date: "2020-03-23"
+source:
+  nom: ARS Martinique
+  url: http://www.martinique.ars.sante.fr/system/files/2020-03/Point%20de%20situation%2023%20mars%202020.pdf
+  archive: https://web.archive.org/web/20200324102201/https://www.martinique.ars.sante.fr/system/files/2020-03/Point%20de%20situation%2023%20mars%202020.pdf
+donneesRegionales:
+  nom: Martinique
+  code: REG-02
+  casConfirmes: 53 # Pages ou pdf de 22/03 et 21/01 non trouvé sur le site ars martinique
+  hospitalises: 7
+  deces: 0
```

Il manque le nombre en réanimation (1).
```suggestion
  reanimation: 1
  deces: 0
```

---

Oups, désolé, j'avais mal lu le communiqué... Ce n'est pas très clair le "(+1)". Et ils précisent "hospitalisés en réanimation". Ne faudrait-il pas mettre une ligne `réanimation` plutôt que `hospitalises` ?

---

> Il manque le nombre en réanimation (1).

Oui merci,
`dont 7 (+1) hospitalisés en réanimation.`  est-ce ça veut dire les 8 sont hospitalisés ou sont en réa?

---

@jdesboeufs ou @Binnette (je ne sais plus) m'avait dit un truc dessus  mais j'ai oublié.